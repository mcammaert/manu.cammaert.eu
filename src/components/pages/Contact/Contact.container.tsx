import React from 'react';

import { getPersonal, getLinks } from 'api/resume';
import Contact from './Contact';

import { ContactContainerProps } from './Contact.types';

const ContactContainer: React.FC<ContactContainerProps> = () => {
  const personal = getPersonal();
  const links = getLinks();
  return <Contact personal={personal} links={links} />;
};

export default ContactContainer;
