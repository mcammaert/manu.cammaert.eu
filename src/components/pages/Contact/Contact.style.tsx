import styled from 'styled-components';

export const Contact = styled.div`
  position: absolute;
`;

export default { Contact };
