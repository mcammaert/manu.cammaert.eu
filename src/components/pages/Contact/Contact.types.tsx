import { Personal, Links } from 'api/resume';

// eslint-disable-next-line @typescript-eslint/no-empty-interface
export interface ContactContainerProps {}

// eslint-disable-next-line @typescript-eslint/no-empty-interface
export interface ContactProps {
  personal: Personal;
  links: Links;
}
