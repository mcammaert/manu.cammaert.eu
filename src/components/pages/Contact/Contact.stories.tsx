import React from 'react';

import { storiesOf } from '@storybook/react';

import Contact from './Contact';

storiesOf('Contact', module).add('Contact', () => {
  return <Contact />;
});
