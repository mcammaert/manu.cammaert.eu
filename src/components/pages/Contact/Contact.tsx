import React from 'react';

import { Default } from 'components/templates/Default';

import { ContactProps } from './Contact.types';

import S from './Contact.style';

const Contact: React.FC<ContactProps> = () => (
  <S.Contact>
    <Default>
      <p>Contact</p>
    </Default>
  </S.Contact>
);

export default Contact;
