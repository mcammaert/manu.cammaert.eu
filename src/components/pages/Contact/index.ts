import * as S from './Contact.style';

export { default as Contact } from './Contact';
export * from './Contact.types';
export { S };
