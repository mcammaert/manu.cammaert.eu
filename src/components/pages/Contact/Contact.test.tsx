import React from 'react';

import { cleanup } from '@testing-library/react';
import 'jest-styled-components';

import { renderComponent } from 'testing/react-testing-library';

import Contact from './Contact';

afterEach(cleanup);

const theme = {
  colors: {},
};

describe('Contact', () => {
  describe('default props', () => {
    it('should match the snapshot', () => {
      const component = <Contact />;
      const { container } = renderComponent({ theme, component });
      expect(container).toMatchSnapshot();
    });
  });
});
