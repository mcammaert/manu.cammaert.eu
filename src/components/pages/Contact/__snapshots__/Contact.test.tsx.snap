// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact default props should match the snapshot 1`] = `
.c1 {
  position: absolute;
}

.c0 {
  position: absolute;
}

<div>
  <div
    class="c0"
  >
    <div
      class=""
    >
      <p>
        Contact
      </p>
      <div
        class="c1"
      >
        Footer
      </div>
    </div>
  </div>
</div>
`;
